PA193_mnemonic_Slytherin/mnemonic.py

```diff
@@ -38,24 +40,47 @@ def __generate_seed(mnemonic: str, seed_password: str = '') -> bytes:
 # - dictionary is too big (2048 lines OK, but too long words) like hundreds of MB...
 # - every line has exactly 1 word (no whitespaces)
 
-def __entropy2mnemonic(entropy: bytes) -> str:
-    """Convert entropy to mnemonic phrase using dictionary.
+def __get_dictionary() -> Tuple[List[str], Dict[str, int]]:
+    """Load the dictionary.
     Currently uses 1 default dictionary with English words.
     # TODO Should we support multiple dictionaries for various languages?
+    # TODO implement soma checks
+    :rtype: Tuple[List[str], Dict[str, int]]
+    :return: List and dictionary of words
+
+    """
+    with open(os.path.join(os.path.dirname(os.path.abspath(__file__)), 'english.txt'), 'r') as f:
```

Comment:
Some checks should be here: what if file wasn't opened, what if it hasn't a structure we need, what if we've read more or less than we need, etc.

Reply:
Same as here, it should raise exception, this function cannot recover from such error, the caller should handle it. https://github.com/mvondracek/PA193_mnemonic_Slytherin/pull/19#discussion_r334272962